README.md

```diff
@@ -350,15 +350,13 @@ On Linux, you can use the portal implementation instead of GTK, which will open
 
 To use the portal implementation, add `-DNFD_PORTAL=ON` to the build command.
 
-*Note:  Setting a default path is not supported by the portal implementation, and any default path passed to NFDe will be ignored.  This is a limitation of the portal API, so there is no way NFDe can work around it.  If this feature is something you desire, please show your interest on https://github.com/flatpak/xdg-desktop-portal/pull/874.*
-
-*Note 2:  The folder picker is only supported on org.freedesktop.portal.FileChooser interface version >= 3, which corresponds to xdg-desktop-portal version >= 1.7.1.  `NFD_PickFolder()` will query the interface version at runtime, and return `NFD_ERROR` if the version is too low.
+*Note:  The folder picker is only supported on org.freedesktop.portal.FileChooser interface version >= 3, which corresponds to xdg-desktop-portal version >= 1.7.1.  `NFD_PickFolder()` will query the interface version at runtime, and return `NFD_ERROR` if the version is too low.
 
 ### What is a portal?
 
 Unlike Windows and macOS, Linux does not have a file chooser baked into the operating system.  Linux applications that want a file chooser usually link with a library that provides one (such as GTK, as in the Linux screenshot above).  This is a mostly acceptable solution that many applications use, but may make the file chooser look foreign on non-GTK distros.
 
-Flatpak was introduced in 2015, and with it came a standardized interface to open a file chooser.  Applications using this interface did not need to come with a file chooser, and could use the one provided by Flatpak.  This interface became known as the desktop portal, and its use expanded to non-Flatpak applications.  Now, most major desktop Linux distros come with the desktop portal installed, with file choosers that fit the theme of the distro.  Users can also install a different portal backend if desired.  There are currently two known backends: GTK and KDE.  (XFCE does not currently seem to have a portal backend.)
+Flatpak was introduced in 2015, and with it came a standardized interface to open a file chooser.  Applications using this interface did not need to come with a file chooser, and could use the one provided by Flatpak.  This interface became known as the desktop portal, and its use expanded to non-Flatpak applications.  Now, most major desktop Linux distros come with the desktop portal installed, with file choosers that fit the theme of the distro.  Users can also install a different portal backend if desired.  There are currently many backends: GTK, Gnome, KDE, or wlroots; see the [desktop portal page](https://flatpak.github.io/xdg-desktop-portal/) for a complete list.
```

Comment:
Hmm, I think I was told that xdg-desktop-portal-wlr doesn't have a file chooser, and so wlroots users need to also install another portal backend to use a file chooser.  Do you know if wlroots really supports it now?

Reply:
My bad, I was too hasty...

From my (slower) understanding (and just for the file chooser part):
* xdg-desktop-portal-wlr still does not support it.
* xdg-desktop-portal-gnome depends on xdg-desktop-portal-gtk to provide it (according to `apt show xdg-desktop-portal-gnome`).
* xdg-desktop-portal-xapp too.
* xdg-desktop-portal-gtk implements its own file chooser.
* xdg-desktop-portal-kde too.
* xdg-desktop-portal-lxqt too.
 
So, many desktop managers support it (my half-baked wording) but the backends providing the file chooser are only Gtk, KDE, and LXQt (the original wording).

If you are fine with it, I will simply reuse the original text and add LXQt as a third backend.

Reply:
Thanks for the elaboration!  Yes, I think we can add LXQt as a third backend, and also adopt your other suggestion to say that Cinnamon, MATE and XFCE can rely on the Xapp backend.